operations/package_repository_management/Nexus_Export_and_Restore.md

```diff
@@ -85,3 +85,22 @@ If the old export is not deleted, then the new job will overwrite the old export
 ```bash
 kubectl delete pvc -n nexus nexus-bak
 ```
+
+### Cleanup failed or stopped export
+
+If an export is stopped prematurely or fails to complete there is a few steps that need to be taken to bring Nexus back into a working state.
+
+1. Delete the failed or stopped job, see [Cleanup export job](#cleanup-export-job)
```

Comment:
do these anchors exist in this (or other file)? (cleanup-export-job and cleanup-previous-export)

Reply:
They are links the the same file. Are they not working for you?

Reply:
They worked for me.

Reply:
sorry -- i forgot the auto-anchor thing.  never mind :)